build.cmd

```diff
@@ -6,10 +6,20 @@ setlocal
 ::       means that that rebuilding cannot successfully delete the task
 ::       assembly.
 
+:ReadArguments
+:: Read in the args to determine whether to run the native build, managed build, or both (default)
+set "__args=%*"
+set "__buildSpec=both"
+if /i [%1] == [both]    	(set __buildSpec=both&&set "__args=%__args:~4%"&&shift&&goto Tools)
```

Comment:
I would just exclude this an explicit option. If it is needed then don't specify either.